packages/react-percy-ci/src/index.js

```diff
@@ -1,6 +1,7 @@
 import ApiClient from '@percy-io/react-percy-api-client';
 import compileAssets from './compileAssets';
 import createDebug from 'debug';
+import { each } from 'bluebird';
```

Comment:
Sure thing, remove `bluebird` here as well as the other place I had already been using it.